plugins/modules/repository_set_info.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021 William Bradford Clark
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: repository_set_info
+version_added: 2.0.0
+short_description: Fetch information about Red Hat Repositories
+description:
+  - Fetch information about Red Hat Repositories
+author: "William Bradford Clark (@wbclark)"
+options:
+  name:
+    description:
+      - Name of the repository set
+    required: false
+    type: str
+  product:
+    description:
+      - Name of the parent product
+    required: false
+    type: str
+  label:
+    description:
+      - Label of the repository set. Can be used in place of I(name) & I(product)
+    required: false
+    type: str
+  repositories:
+    description:
+      - Release version and base architecture of the repositories
+      - Some repository sets require only I(basearch) or only I(releasever)
+      - Required when I(all_repositories) is unset or C(false)
+    required: false
+    type: list
+    elements: dict
+    suboptions:
+      basearch:
+        description:
+          - Basearch of the repository
+        type: str
+      releasever:
+        description:
+          - Releasever of the repository
+        type: str
+  all_repositories:
+    description:
+      - Find all repositories in the repository set
+      - Required when I(repositories) is unset or an empty list
+    required: false
+    type: bool
+  enabled:
+    description:
+      - When true, only return repository sets which are already enabled. Defaults to false.
```

Comment:
```suggestion
      - When true, only return repository sets which are already enabled.
```

We have `default: false` below already.